### PR TITLE
Fix TestCase HTTP related deprecation warnings

### PR DESCRIPTION
**Pull request by tute**
Fixes issue(s) #228.

Changes proposed in this pull request:

- Fixes TestCase HTTP methods deprecation warning

/cc @jessieay.

### CODE REVIEW

**Comment by afeld** [comment]
Well done, old chap 👏

File: spec/controllers/broadcast_messages_controller_spec.rb [hunk 1]
```diff
@@ -18,7 +18,9 @@
       user = create(:user, admin: false)
       allow(controller).to receive(:current_user).and_return(user)
 
-      post :create, broadcast_message: { title: "t", body: "b" }
+      process :create, method: :post, params: {
```
Comment (jessieay) [inline spec/controllers/broadcast_messages_controller_spec.rb]: waaat this is cool

**Comment by jessieay** [comment]
Oy this did not auto-close the related issue. Opened an issue for that https://github.com/18F/dolores-landingham-slack-bot/issues/237

**Comment by afeld** [comment]
Though to be fair, this pull request didn't actually address #228 😉

**Comment by jessieay** [comment]
you are right! you almost tricked me

**Pull request merged**